src/localisation/launch/ekf_node.launch

```diff
@@ -0,0 +1,68 @@
+<launch>
+    <node name="ekf_node" pkg="localisation" type="ekf_node" output="screen">
+	<!-- The standard deviation of the IMU angular velocity measurement -->
+	<!-- Put in the error of the IMU (angular velocity) -->
+        <rosparam param="sdev_gyro">0.017453292519943295</rosparam>
+
+	<!-- The standard deviation of the linear (forward) speed encoder measurement ->>
```

Comment:
Ending of this line should be `-->`, not `->>` (invalid syntax and won't launch otherwise)